README.md

```diff
@@ -36,11 +36,27 @@ How To (Plugin Developers)
 <dependency>
     <groupId>com.destroystokyo.paper</groupId>
     <artifactId>paper-api</artifactId>
-    <version>1.15.2-R0.1-SNAPSHOT</version>
+    <version>1.16.1-R0.1-SNAPSHOT</version>
     <scope>provided</scope>
  </dependency>
  ```
+#####Or alternatively, with Gradle:
```

Comment:
Is this meant to be a header?

Reply:
Oh my bad, forgot the space haha. Think you're able to edit that if you're a maintainer.

Reply:
IntelliJ sees that as a header for some reason, that's the issue. In my original I had the space in there, but remade it and forgot about it this time.

Reply:
Okay fixed the header in the latest commit, now has proper spacing.